model/Dataset/Classes/Dataset.md

```diff
@@ -68,3 +66,32 @@ Metadata information that can be added to a dataset that may be used in a softwa
   - type: DatasetAvailabilityType
   - minCount: 0
   - maxCount: 1
+
+## External properties restrictions
+
+- /Core/Element/name
+  - minCount: 1
+- /Core/Properties/originatedBy
+  - minCount: 1
+- /Core/Properties/suppliedBy
+  - minCount: 0
+- /Core/Hash
+  - minCount: 0
+- /Software/Package/downloadLocation
+  - minCount: 1
+- /Software/Package/packageVersion
+  - minCount: 0
+- /Software/Properties/purpose
+  - minCount: 1
+- /Core/Element/comment
+  - minCount: 0
+- /Licensing/Properties/concludedLicense
```

Comment:
ConcludedLicense and licensing is not yet in core and this can not yet be merged. This also might be expressed as a relationship, so this approach might not work.

Reply:
The issue is that it references a property. And even with licensing profile merged, one can not add a constraint to a field that does not yet exist